### ISSUE TITLE
Visualizing loss in tensorboard

### ISSUE THREAD

**Issue by PabloVD** [lucidrains/denoising-diffusion-pytorch#194]
Is it possible to visualize evolution of loss using tensorboard when training with `Trainer()`?
Thanks!
